game_api.txt

```diff
@@ -155,7 +155,7 @@ The doors mod allows modders to register custom doors and trapdoors.
 ### Fence gate definition
 
 	description = "Wooden Fence Gate",
-	texture = "default_wood.png",
+	texture = "default_wood.png", -- `backface_culling` will automatically be set to `true` if it's not given.
```

Comment:
Code style is wrong in many places, but the rule is 90 column (tabsize 4) hard-limit, 80 column soft-limit. So if possible split at 80 but occasionally if much clearer up to 90 is allowed.

No need for -- ^ comments are understood to refer to the line below, and here it is obvious it refers to the texture definition.